src/chainstate/stacks/boot/bns.clar

```diff
@@ -878,6 +878,9 @@
           (> block-height (+ lifetime lease-started-at)) 
           (<= block-height (+ (+ lifetime lease-started-at) NAME_GRACE_PERIOD_DURATION)))))))
 
+(define-read-only (get-name (owner principal))
+  (ok (map-get? owner-name owner)))
```

Comment:
Since this is a `define-read-only` function, this shouldn't need to return a response (especially since there's no error case here) -- can this function instead just return `(map-get? owner-name owner)`?

Reply:
All the other read-only functions return a response value. 

Alternatively, I suggest to return `ERR_NAME_NOT_FOUND` in case of an "owner" without a name:
`(match (map-get? owner-name owner) username (ok username) (err ERR_NAME_NOT_FOUND))`

Reply:
Yep, I think using `ERR_NAME_NOT_FOUND` would be better.